indico/migrations/versions/20200619_1526_b6dd0a4ed40d_add_registration_form_to_event_acls.py

```diff
@@ -0,0 +1,32 @@
+"""Add registration form to event acls
+
+Revision ID: b6dd0a4ed40d
+Revises: c0fc1e46888b
+Create Date: 2020-06-19 15:26:01.961716
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'b6dd0a4ed40d'
+down_revision = 'c0fc1e46888b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('principals', sa.Column('registration_form_id', sa.Integer(), nullable=True), schema='events')
+    op.create_index(op.f('ix_principals_registration_form_id'), 'principals', ['registration_form_id'], unique=False, schema='events')
+    op.create_foreign_key(op.f('fk_principals_registration_form_id_forms'), 'principals', 'forms', ['registration_form_id'], ['id'], source_schema='events', referent_schema='event_registration')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(op.f('fk_principals_registration_form_id_forms'), 'principals', schema='events', type_='foreignkey')
+    op.drop_index(op.f('ix_principals_registration_form_id'), table_name='principals', schema='events')
+    op.drop_column('principals', 'registration_form_id', schema='events')
+    # ### end Alembic commands ###
```

Comment:
Remove the comments and cleanup the revision like we do in other cases (e.g. not specifying names that are implied anyway if omitted, no `op.f()`, etc)